src/packages/core/extension-registry/conditions/section-alias.condition.tempts

```diff
@@ -0,0 +1,60 @@
+import { UmbBaseController, UmbControllerHost } from '@umbraco-cms/backoffice/controller-api';
```

Comment:
What does this file do?

Reply:
Oh, yes. that was a version that I left behind when I realized that we most likely do not need the ability for Context APIs to unprovide them selves. Cause we don't have that case currently, and I think we can continue with out. Cause as seen by the amount of code in this file, which complicates things.

I'll delete it.